mocks/landscape/landscapemockservice/landscape_mock_service.go

```diff
@@ -263,6 +266,21 @@ func (s *Service) IsConnected(uid string) bool {
 	return s.isConnected(uid)
 }
 
+// WaitDisconnection returns a channel that will be closed when the connection with the host assigned to the
+// specified UID is terminated.
+//
+// If the UID is not registered, the second return value will be false.
+func (s *Service) WaitDisconnection(uid string) <-chan struct{} {
```

Comment:
I don't see this being used outside of tests. Is it for future PR's?

Reply:
Well it is a mock Landscape server, so it's only gonna be used it tests :P